Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -876,5 +876,33 @@ public static void BarberShop(Session session, params string[] parameters)
             chain.AddAction(session.Player, () => session.Network.EnqueueSend(new GameEventStartBarber(session)));
             chain.EnqueueChain();
         }
+        /// <summary>
+        /// Debug console command to test the GetSpellFormula function.
+        /// </summary>
+        [CommandHandler("spell", AccessLevel.Admin, CommandHandlerFlag.ConsoleInvoke, 0, "Tests spell formula calculation")]
```

Comment:
The AccessLevel.Admin was just a holdover since I'd originally had this be an in-game function and moved it. Was more practical to move to console (so I didn't need to log in). Adjusted that AccessLevel (doubt it's even required for a console command) to be consistent with the rest and renamed the function.